source/_docs/guides/drupal-8-composer-no-ci.md

```diff
@@ -0,0 +1,242 @@
+
+First, clone the `example-drops-8-composer` repository locally.
+
+```
+git clone git@github.com:pantheon-systems/example-drops-8-composer.git $PANTHEON_SITE_NAME
+```
+
+Change into the cloned directory.
+
+```
+cd $PANTHEON_SITE_NAME
+```
+
+## Updating the git remote URL
+
+Store the git URL for the Pantheon site created earlier in a variable.
+
+```
+PANTHEON_SITE_GIT_URL="$(terminus connection:info $PANTHEON_SITE_NAME.dev --field=git_url)"
+```
+
+Update the git remote to use the Pantheon site git URL returned rather than the `example-drops-8-composer` GitHub URL.
+
+```
+git remote set-url origin $PANTHEON_SITE_GIT_URL
+```
+
+## Removing Automation Pieces
+`example-drops-8-composer` was designed to run automated tests on a continuous integration server. Since contonuous integration won't be used you can safely remove the directories below.
+
+* `scripts/github`
+* `scripts/gitlab`
+* `.circleci`
+
+If you don't plan on running automated tests locally you can completely remove the testing functionality.
+
+<div class="panel panel-drop panel-guide" id="accordion">
+  <div class="panel-heading panel-drop-heading">
+    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#unique-anchor">
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span>Remove Test Suites</h3>
+    </a>
+  </div>
+  <div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
+    First, delete the `tests` directory. Next, you will need to modify `composer.json`.
+
+    * Remove all dependencies in the `require-dev` section
+    * Update the `scripts` section to the following:
+
+    ```
+        "scripts": {
+            "build-assets": [
+                "@prepare-for-pantheon",
+                "composer install --optimize-autoloader --no-dev"
+            ],
+            "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+            "prepare-for-pantheon": "DrupalProject\\composer\\ScriptHandler::prepareForPantheon",
+            "post-install-cmd": [
+                "@drupal-scaffold",
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ],
+            "post-update-cmd": [
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ],
+            "post-create-project-cmd": [
+                "@drupal-scaffold",
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ]
+        },
+    ```
+  </div>
+</div>
+
+## Managing Drupal with Composer
+
+### Downloading Drupal Dependencies with Composer
+
+Normally the next step would be going through the standard Drupal installation. But since we’re using Composer, none of the core files exist yet. Let’s use Composer to install Drupal core.
```

Comment:
Can we change this to `Let’s use Composer to download Drupal core.` since `installation` was used in the previous sentence to mean the database operation of installing core and here we mean downloading files.